Examples/VisionExamples/Examples/ARCustomizationViewController.swift

```diff
@@ -0,0 +1,62 @@
+import MapboxVisionAR
+import UIKit
+
+/**
+ * "AR Customization" tutorial demonstrates how to customize vision AR visuals - AR lane and AR fence.
+ * The tutorial is based on "ARNavigation" example which shows how to get the route and set up Vision AR session.
+ */
+class ARCustomizationViewController: ARNavigationViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Adjust AR rendering quality
+        visionARViewController.set(arQuality: 0.8)
+
+        // Make sure lane is visible
+        visionARViewController.isLaneVisible = true
+        // Set lane length in meters
+        visionARManager.set(laneLength: 40)
+
+        // Create an instance of LaneVisualParams
```

Comment:
nit:

```suggestion
        // Create an instance of `LaneVisualParams`
```
to reflect the comments about the fence below

Reply:
Agree.